src/styles/blocks/card.scss

```
@@ -0,0 +1,113 @@
+.card {
+  width: 200px;
+  display: flex;
+  box-sizing: border-box;
+  flex-direction: column;
+  align-items: center;
+
+  border: 1px solid $border-color;
+  border-radius: 5px;
+
+  &__image {
+    width: 160px;
+    height: 134px;
+
+    margin-top: 32px;
+  }
+
+  &__title {
+    font-size: 12px;
+    font-weight: 500;
+    line-height: 18px;
+    color: $font-main-color;
+
+    text-align: left;
+
+    height: 36px;
+    width: 166px;
+    margin-top: 40px;
+    margin-bottom: 0;
+  }
+
+  &__description {
+    text-align: left;
+    width: 166px;
+    height: 14px;
+    line-height: 14px;
+    color: $font-secondary-color;
+
+    margin-top: 4px;
+    margin-bottom: 0;
+  }
+
+  &__reviews {
+    margin-top: 16px;
+    margin-bottom: 24px;
+
+    width: 166px;
+    height: 16px;
+
+    line-height: 14px;
+    text-align: right;
+
+    color: $font-main-color;
+
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+  }
+
+  &__price {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+
+    height: 18px;
+    width: 166px;
+
+    margin: 0;
+
+    &-value {
+      font-size: 16px;
+      font-weight: 700;
+      line-height: 18px;
+    }
+
+    &-title {
+      font-size: 12px;
+      font-weight: 400;
+      line-height: 18px;
+      color: $font-secondary-color;
+    }
+  }
+
+  &__button {
+    text-decoration: none;
+    color: $card-main-color;
+    width: 166px;
```

```suggestion
    width: 100%;
```
Consider setting paddings for the card instead of hardcoding width